plugins/mac-videotoolbox/encoder.c

```diff
@@ -344,6 +352,60 @@ create_encoder_spec(const char *vt_encoder_id)
 
 	return encoder_spec;
 }
+
+static inline CFMutableDictionaryRef
+create_prores_encoder_spec(CMVideoCodecType target_codec_type,
+			   bool hardware_accelerated)
+{
+	CFStringRef encoder_id = NULL;
+
+	CFArrayRef encoder_list;
+	VTCopyVideoEncoderList(NULL, &encoder_list);
+	CFIndex size = CFArrayGetCount(encoder_list);
+
+	for (CFIndex i = 0; i < size; ++i) {
+		CFDictionaryRef encoder_dict =
+			CFArrayGetValueAtIndex(encoder_list, i);
+
+		if (__builtin_available(macOS 10.14, *)) {
```

Comment:
@gxalpha Mark for refactoring later: Lowest supported MacOS version is 10.15 now, so we can assume this to be available.